pkgs/development/libraries/lucene++/default.nix

```diff
@@ -17,6 +17,14 @@ stdenv.mkDerivation rec {
   postPatch = ''
     substituteInPlace src/test/CMakeLists.txt \
       --replace "add_subdirectory(gtest)" ""
+
+    substituteInPlace src/config/core/liblucene++.pc.in \
+      --replace "libdir=@LIB_DESTINATION@" "libdir=\''${prefix}/lib" \
+      --replace "-L@LIB_DESTINATION@" "-L\''${libdir}"
+
+    substituteInPlace src/config/contrib/liblucene++-contrib.pc.in \
+      --replace "libdir=@LIB_DESTINATION@" "libdir=\''${prefix}/lib" \
+      --replace "-L@LIB_DESTINATION@" "-L\''${libdir}"
```

Comment:
Is this fixed upstream and we could fetch a commit?

Reply:
It has fixed in https://github.com/luceneplusplus/LucenePlusPlus/commit/39cd44bd54e918d25ee464477992ad0dc234dcba

And header files (https://github.com/NixOS/nixpkgs/pull/171009) should fixed in https://github.com/luceneplusplus/LucenePlusPlus/commit/ad09039199eca46183f1c4794e531df62491bb48 ,but it can't apply to [rel_3.0.8](https://github.com/luceneplusplus/LucenePlusPlus/releases/tag/rel_3.0.8) version （missing multiple commits）